docs/examples.md

```diff
@@ -36,8 +36,8 @@ CREATE STREAM pageviews \
   (viewtime BIGINT, \
    userid VARCHAR, \
    pageid VARCHAR) \
-  WITH (kafka_topic='pageviews-topic', \
-        value_format='DELIMITED');
+  WITH (KAFKA_TOPIC='pageviews-topic', \
```

Comment:
We also switched e.g. KSQL decks to use uppercase for properties in the WITH clause.